### DISCUSSION TITLE
SW Update and OTA

### DISCUSSION THREAD

**Discussion by aurang-microchip** [polarfire-soc/polarfire-soc-documentation#104] [Linux build systems for PolarFire SoC] [answered]
Hi,

I just wanted to ask about the Support for Software Update (SW update or swu) and OTA (On-the-Air) Update as we came across a question.
According to my knowledge, SW update should work with PolarFire SoC. Example, a recipe added to the Yocto Build System.
https://github.com/sbabic/swupdate
Has anyone in the Development Team tried to use it with PFSoC?

Second question is regarding OTA Update, do we support it. If Yes, has someone tried it and how it could be implemented.

Thanks!
Best Regards,
Aurang Zaib

**Comment by hughbreslin** [comment, marked as answer]
Hi @aurang-microchip, thanks for sharing this repo, its not one we've looked at before. 

SWU / OTA updates aren't something we've looked at in detail yet, we can update this post when we have more info :)
